app/controllers/payments_controller.rb

```diff
@@ -0,0 +1,53 @@
+class PaymentsController < ApplicationController
+  load_and_authorize_resource
+  before_action :authenticate_user!
+  before_action :find_user
+
+  def index
+    @group_uniq = Group.find(params[:group_id])
+    @payments = Payment.where(author_id: @user.id).order(created_at: :desc)
+    @group = Payment.order(created_at: :desc).joins(:groups).pluck(:'groups.icon')
```

Comment:
- [ ] You are supposed to display transactions(payments) under specific categories. Filtered with `group_id`. Please kindly fix it

Reply:
hi @m05e5 

please give more explanation, because I believe this is exactly how the app works now 

![payments](https://user-images.githubusercontent.com/70941894/230738412-73fa0348-828e-47e3-b85e-ed649553221f.PNG)